src/BinSkim.Driver/MultithreadedAnalyzeCommand.cs

```diff
@@ -29,6 +30,21 @@ public override IEnumerable<Assembly> DefaultPluginAssemblies
             set => throw new InvalidOperationException();
         }
 
+        protected override bool ShouldComputeHashes(string file, BinaryAnalyzerContext context)
+        {
+            return base.ShouldComputeHashes(file, context)
+                && IsValidScanTarget(file);
```

Comment:
>&&  [](http://example.com/codeflow?start=16&length=3)

nit: move the && operator to end of previous line. Or move the || operator in next function to start of next line.